### PR TITLE
1892 compose processor unit tests

### LINKED ISSUE #1892: Implement unit tests for pipeline element ‘ComposeProcesor‘
This issue is about implementing unit tests for our pipeline element [ComposeProcessor](https://github.com/apache/streampipes/blob/752289c6acfc8fa10398c920d678c5c02143ed5c/streampipes-extensions/streampipes-processors-filters-jvm/src/main/java/org/apache/streampipes/processors/filters/jvm/processor/compose).
An example of how to write unit tests for pipeline elements can be found [here](https://github.com/apache/streampipes/tree/ca4e0c2d44c1a713caaee3e9c1bff46096f41c53/streampipes-extensions/streampipes-processors-transformation-jvm/src/test/java/org/apache/streampipes/processors/transformation/jvm/processor).

### Mentoring
As this issue is marked as a good first issue and for the `hacktoberfest` event: one of @dominikriemer, @tenthe, @RobertIndie, or @bossenti are happy to provide help for getting started, just tag (one of) them if you want to start working on this issue and need some help.

### PR DESCRIPTION
**Pull request by IsaakKrut**
Purpose
Closes #1892 

Remarks
PR introduces (a) breaking change(s): no

PR introduces (a) deprecation(s): no

### CODE REVIEW

File: streampipes-extensions/streampipes-processors-filters-jvm/src/test/java/org/apache/streampipes/processors/filters/jvm/processor/compose/TestComposeProcessor.java [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+    });
+  }
+
+
+
+  @Test
+  public void testComposeProcessor() {
+    LOG.info("Executing test: {}", testName);
+    ComposeProcessor processor = new ComposeProcessor();
+    DataProcessorDescription originalGraph = processor.declareModel();
```
Comment (tenthe) [inline streampipes-extensions/streampipes-processors-filters-jvm/src/test/java/org/apache/streampipes/processors/filters/jvm/processor/compose/TestComposeProcessor.java:112]:
```suggestion
    var originalGraph = processor.declareModel();
```

**Comment by tenthe** [comment]
@IsaakKrut thank you very much for providing this PR.
I have a small comment regarding the code style. We have started to switch to the new `var` syntax in Java. I have adapted it to an example in your code.
Can you please adjust your code to use this.

Thank you very much for your effort.

**Comment by IsaakKrut** [comment]
@tenthe updated the PR

**Review by tenthe — APPROVED** [review]

**Review by dominikriemer — APPROVED** [review]

**Comment by dominikriemer** [comment]
Thanks @IsaakKrut !

**Pull request merged**